### ISSUE TITLE
Tearing after automated preset load, even with Clock Gen correctly installed

### ISSUE THREAD

**Issue by Em-K-IronFist** [ramapcsx2/gbs-control#130]
I noticed a small inconsistency with vsync when using the board with the Clock Generator installed.

On boot up, whatever the first format is will sync correctly and appear as expected; however, upon format change, a small horizontal tear will appear at a fixed location on the screen; this tear doesn't look like standard vsync tearing, but has more of a jagged quality to it. It's also not as noticeable as a normal tearing. 

For example, if I'm playing something in 480p, turn it off, and load up a game in 240p, there will usually be this fixed tear line. This can be worked around by pressing "Load Preset" on the web interface, or resetting the arduino, or powercycling the whole thing. which in my experience seems to reset everything enough that sync timing is read correctly again and the tear is no longer visible.

**Comment by ramapcsx2** [comment]
Yeah, this can happen, as there is no mechanism to drive the tear line off screen.
I simply rely on it most often landing there (due to some hidden mechanism).
I still want to find a way to have control over the process.
